packages/modelviewer.dev/examples/examples-and-docs.html

```diff
@@ -0,0 +1,216 @@
+  <script src="../node_modules/focus-visible/dist/focus-visible.js" defer></script>
+
+  <style>
+    html {
+      scroll-behavior: smooth;
+    }
+
+    .sidebar {
+      margin: 0;
+      padding: 0;
+      width: 300px;
+      background-color: #f6f6f6;
+      position: fixed;
+      height: 100%;
+      overflow: auto;
+      border: 1px solid #BDBDBD;
+    }
+
+    .outer-content {
+      margin-left: 300px;
+    }
+
+    .inner-content {
+      padding: 10px 40px;
+    }
+
+    p {
+      display: inline;
+    }
+
+    .attribute-name {
+      padding: 10px 20px 10px 20px;
+      margin: 10px 0 20px 0;
+      background-color: #ececec;
+    }
+
+    .attribute-definition {
+      margin: 0 0 20px 0;
+    }
+
+    .category-container {
+      padding-bottom: 20px;
+    }
+
+    .attribute-container {
+      padding-bottom: 20px;
+    }
+
+    .value-table {
+      margin: 0 0 30px 0;
+      width: 50%;
+    }
+
+    table,
+    td,
+    th {
+      border: 1px solid #ececec;
+      border-collapse: collapse;
+    }
+
+    th {
+      font-weight: bold;
+    }
+
+    td:hover {
+      background-color: #ececec;
+    }
+
+    ul.links {
+      margin-top: 5px;
+    }
+
+    hr.solid {
+      border-top: 3px solid #f6f6f6;
+    }
+
+    .header {
+      background-color: #f6f6f6;
+      padding: 10px 20px 10px 40px;
+      width: 100%;
+      position: sticky;
+      top: 0;
+      border-top: 1px solid #BDBDBD;
+      border-bottom: 1px solid #BDBDBD;
+    }
+  </style>
+</head>
+
+<body>
+  <div class="sidebar">
+    <ol>
+      <li><a href="#introduction">Introduction</a></li>
+      <li><a href="#loading">Loading</a></li>
+      <li><a href="#animation">Animation</a></li>
+      <li><a href="#lighting-and-environment">Lighting &amp; Environment</a></li>
+      <li><a href="#model-formats">Model Formats</a></li>
+      <li><a href="#staging-and-camera-controls">Staging &amp; Camera Controls</a></li>
+      <li><a href="#augmented-reality">Augmented Reality</a></li>
+    </ol>
+  </div>
+
+  <!-- Start of all documentation -->
+  <div class="outer-content">
+    <div id="loading-docs"></div>
+    <div id="animation-docs"></div>
+  </div>
+
+  <script>
+    async function fetchHtmlAsText(url) {
+      const response = await fetch(url);
+      return await response.text();
+    }
+
+    async function load() {
+      const contentDivLoading = document.getElementById("loading-docs");
+      contentDivLoading.innerHTML = await fetchHtmlAsText("./docs/loading-docs.html");
+
+      const contentDivAnimation = document.getElementById("animation-docs");
+      contentDivAnimation.innerHTML = await fetchHtmlAsText("./docs/animation-docs.html");
+    }
+
+    load()
+  </script>
+
+  <script>
+    window.onscroll = function () { myFunction() };
+
+    function myFunction() {
+      var headers = document.getElementsByClassName("header");
+      for (var i = 0; i < headers.length; i++) {
```

Comment:
I think you can just do `for(const header : headers)`

Reply:
This begs another question, which is how well do these pages work on different browsers? We generally try to push for the adoption of modern web techniques (like `const` and `let` instead of `var`), which now are pretty wide-spread, with the usual exception of IE11. I think there's a problem with our current pages on IE11 anyway, which I haven't been able to fix due to not being able to repro. So maybe it doesn't matter, but let's at least keep it in the back of your mind.